plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/CodeWhispererFeatureConfigService.kt

```diff
@@ -82,6 +83,17 @@ class CodeWhispererFeatureConfigService {
 
     fun getChatWSContext(): Boolean = getFeatureValueForKey(CHAT_WS_CONTEXT).stringValue() == "TREATMENT"
 
+    // convert into mynahUI parsable string
+    // format: '[["key1", {"name":"Feature1","variation":"A","value":true}]]'
+    fun getFeatureConfigJsonString(): String {
+        val jsonString = featureConfigs.entries.map { (key, value) ->
+            "[\"$key\",${Gson().toJson(value)}]"
+        }
+        return """
```

Comment:
can we return jsonstring directly?

Reply:
No it will fail. I spent 6 hours making it correct.

Reply:
It is trying to deserialize as a typescript MAP on MynahUI end